hibernate-reactive-core/src/test/java/org/hibernate/reactive/ORMReactivePersistenceTest.java

```diff
@@ -0,0 +1,187 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.reactive.mutiny.Mutiny;
+import org.hibernate.reactive.provider.Settings;
+import org.hibernate.reactive.stage.Stage;
+import org.hibernate.reactive.testing.DatabaseSelectionRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import io.vertx.ext.unit.TestContext;
+
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
+import static org.hibernate.reactive.util.impl.CompletionStages.completedFuture;
+
+public class ORMReactivePersistenceTest extends BaseReactiveTest {
+
+	@Rule
+	public DatabaseSelectionRule rule = DatabaseSelectionRule.runOnlyFor( POSTGRESQL );
+
+	private SessionFactory ormFactory;
+
+	@Override
+	protected Configuration constructConfiguration() {
+		Configuration configuration = super.constructConfiguration();
+		configuration.addAnnotatedClass( Flour.class );
+		return configuration;
+	}
+
+	@Before
+	public void prepareOrmFactory() {
+		Configuration configuration = constructConfiguration();
+		configuration.setProperty( Settings.DRIVER, "org.postgresql.Driver" );
+		configuration.setProperty( Settings.DIALECT, "org.hibernate.dialect.PostgreSQL95Dialect");
+
+		StandardServiceRegistryBuilder builder = new StandardServiceRegistryBuilder()
+				.applySettings( configuration.getProperties() );
+
+		StandardServiceRegistry registry = builder.build();
+		ormFactory = configuration.buildSessionFactory( registry );
+	}
+
+	@After
+	public void cleanDb(TestContext context) {
+		ormFactory.close();
```

Comment:
If an error happens here the db doesn't get cleaned. Maybe we should add a try catch. I think we can ignore the error print it (if an error occurs here is probably due to something else).

Reply:
print as in....      **context.fail(exception);**     ?

Reply:
Actually, forget about it. We clean the data when we re-run the tests so it's fine the way it is